contrib/ci/fwupd_setup_helpers.py

```diff
@@ -58,9 +58,12 @@ def test_markdown(debug):
 
 
 def test_meson(debug):
-    from importlib.metadata import version
+    from importlib.metadata import version, PackageNotFoundError
 
-    new_enough = version("meson") >= MINIMUM_MESON
+    try:
+        new_enough = version("meson") >= MINIMUM_MESON
+    except PackageNotFoundError:
+        new_enough = False
```

Comment:
Could we call into the system meson and get the version? It seems a bit antisocial to just ignore the distro version.

Reply:
I agree but I'm not sure how to. Any ideas?

Reply:
Would `meson --version` work?

Reply:
Yeah I think I got that setup now, take a look.